src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -229,6 +229,7 @@ void IrcMessageHandler::handleWhisperMessage(Communi::IrcMessage *message)
             app->twitch.server->mentionsChannel->addMessage(_message);
         }
 
+        _message->flags.set(MessageFlag::DoNotTriggerNotification);

```

Comment:
I think this needs to be moved one line down. If I understand correctly, this would also make the whispers tab not getting highlighted. But we only want this for all other channels processed in the forEachChannel-Loop later on.

Correct/Ignore me if I'm being stupid again :)